### PR TITLE
Disable eventd and rsyslog plugin in slim images

### PR DESCRIPTION
**Pull request by zbud-msft**
#### Why I did it

Disable eventd at buildtime for slim images

##### Work item tracking
- Microsoft ADO **(number only)**:26386286

#### How I did it

Add flags for disabling eventd and only copy rsyslog conf files when eventd is included and not slim image

#### How to verify it

Manual testing

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211
- [x] 202305

#### Tested branch (Please provide the tested image version)

- [ ] 
- [ ]

### CODE REVIEW

File: Makefile.work [hunk 1]
```diff
@@ -556,6 +556,7 @@ SONIC_BUILD_INSTRUCTION :=  $(MAKE) \
                            SONIC_CONFIG_USE_NATIVE_DOCKERD_FOR_BUILD=$(SONIC_CONFIG_USE_NATIVE_DOCKERD_FOR_BUILD) \
                            SONIC_INCLUDE_SYSTEM_TELEMETRY=$(INCLUDE_SYSTEM_TELEMETRY) \
                            SONIC_INCLUDE_SYSTEM_GNMI=$(INCLUDE_SYSTEM_GNMI) \
+			   SONIC_INCLUDE_SYSTEM_EVENTD=$(INCLUDE_SYSTEM_EVENTD) \
```
Comment (qiluo-msft) [inline Makefile.work:559]: >			    [](http://example.com/codeflow?start=0&length=6)

Could you use spaces instead of tabs?

File: slave.mk [hunk 2]
```diff
@@ -423,6 +427,7 @@ $(info "INCLUDE_MGMT_FRAMEWORK"          : "$(INCLUDE_MGMT_FRAMEWORK)")
 $(info "INCLUDE_ICCPD"                   : "$(INCLUDE_ICCPD)")
 $(info "INCLUDE_SYSTEM_TELEMETRY"        : "$(INCLUDE_SYSTEM_TELEMETRY)")
 $(info "INCLUDE_SYSTEM_GNMI"             : "$(INCLUDE_SYSTEM_GNMI)")
+$(info "INCLUDE_SYSTEM_EVENTD"             : "$(INCLUDE_SYSTEM_EVENTD)")
```
Comment (qiluo-msft) [inline slave.mk:430]: >   [](http://example.com/codeflow?start=41&length=2)

Could you align the columns?

File: dockers/docker-config-engine-bullseye/Dockerfile.j2 [hunk 3]
```diff
@@ -52,6 +52,7 @@ COPY ["files/readiness_probe.sh", "/usr/bin/"]
 COPY ["files/container_startup.py", "/usr/share/sonic/scripts/"]
 COPY ["00-load-omprog.conf", "/etc/rsyslog.d/"]
 
+
 ## Clean up
```
Comment (qiluo-msft) [inline dockers/docker-config-engine-bullseye/Dockerfile.j2:56]: No need to change.

**Comment by liushilongbuaa** [comment]
> /azpw ms_conflict

@zbud-msft , you need complete the PR first. Then comment.

File: Makefile.work (hunk 1 above)
Comment (zbud-msft) [reply Makefile.work]: Done

File: slave.mk (hunk 2 above)
Comment (zbud-msft) [reply slave.mk]: Done

File: dockers/docker-config-engine-bullseye/Dockerfile.j2 (hunk 3 above)
Comment (zbud-msft) [reply dockers/docker-config-engine-bullseye/Dockerfile.j2]: Done

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**